scripts/winesapos-setup.sh

```diff
@@ -309,6 +309,47 @@ if [ $? -eq 0 ]; then
     sudo swapon /swap/swapfile
     echo "/swap/swapfile    none    swap    defaults    0 0" | sudo tee -a /etc/fstab
     qdbus ${kdialog_dbus} /ProgressDialog org.kde.kdialog.ProgressDialog.close
+elif [ $? -eq 1 ]; then
+    # If the user does not want swap, ask them about zram instead.
+    kdialog --title "winesapOS First-Time Setup" --yesno "Do you want to setup zram instead (recommended if you don't use swap)?"
+    if [ $? -eq 0 ]; then
+        kdialog_dbus=$(kdialog --title "winesapOS First-Time Setup" --progressbar "Please wait for zram to be enabled..." 2 | cut -d" " -f1)
+        qdbus ${kdialog_dbus} /ProgressDialog Set org.kde.kdialog.ProgressDialog value 1
+        # zram half the size of RAM.
+        winesap_ram_size=$(free -m | grep Mem | awk '{print $2}')
+        zram_size=$(expr ${winesap_ram_size} / 2)
+        sudo touch /etc/systemd/system/winesap-zram.service /etc/winesapos/zram-setup.sh
```

Comment:
The script should be located at `/usr/local/bin/winesapos-zram-setup.sh`.

I like to prefix all files we create and manage with the word "winesapos" to make it easier to (1) find and (2) delete if a user wants to uninstall winesapOS bits and switch to a more vanilla and basic Arch Linux installation. Although with your original intention of putting it in `/etc/winesapos/` I do see how `winesapos-zram-setup.sh` would be repetitive there.

Reply:
Alright, I relocated and update the name accordingly